flathub/org.gnome.Boxes
# Delete DConf Hole

Unlike the prior PR #47, this adds the proper path for the gschema. Fixes #45.

Had to use 3.38.0 as 3.38.1 would not compile due to the Tracker1 to Tracker3 migration.

Tested: 
- First login appears only on first run
- Window size is maintained
- Create a Fedora 32 box
- Enter said box
